### ISSUE TITLE
Parabolic Sar - feature-request

### ISSUE THREAD

**Issue by TravelTrader** [bukosabino/ta#51]
First: **GREAT LIBRARY!!**, @bukosabino!
_(better than all the others!!)_

And because of that I would be happy to find the **parabolic sar** from you in it. It's very complicated. I will try to code it for myself. If I'm ready with a good solution I will send it to you. But maybe you're faster! ;)

**Comment by TravelTrader** [comment]
Does anybody know how to code an exact parabolic SAR?
Because I found examples leading to wrong results...

**Comment by bukosabino** [comment]
Hi @TravelTrader ,

When I have time, I will try to implement the indicator based on this explication: https://school.stockcharts.com/doku.php?id=technical_indicators:parabolic_sar

Best,
Dario

**Comment by TravelTrader** [comment]
Thanks @bukosabino!

The reason I am asking the second time is that you're library is the best I found. Your ATR for example is the most exact version I ever found. And for Parabolic SAR I tried all examples and libraries I found online over the last 7 months and in addition to that I tried to code it for myself.
But:
All examples were totally different from the Parabolic SAR of TOS, Yahoo Finance, IB, TradingView, ...

So, I am sure, your version would be the first function I can use for calculations, backtests and charting with Python!
I would be very happy!! :smiley:
